### PR TITLE
Add Aircraft Loadout Dialog

### PR DESCRIPTION
**Pull request by 654wak654**
**When merged this pull request will:**
- This: http://imgur.com/a/dkP4i

**TODO List:**
- [ ] Blacklist / Whitelist functionality.
- [ ] Whatever the fuck that'll please the `rearm` overlords get this merged before ACE 3.15.0
- [ ] Test locality.

### CODE REVIEW

File: addons/aircraft/functions/fnc_onButtonApply.sqf [hunk 1]
```diff
@@ -1,4 +1,18 @@
-// TODO: Header
+/*
+ * Author: 654wak654
+ * Applies the current configuration of pylons to the aircraft
```
Comment (jonpas) [inline addons/aircraft/functions/fnc_onButtonApply.sqf]: Missing final dot (all functions).

**Comment by jonpas** [comment]
Has been decided to add settings and add better integration with the `rearm` component. @654wak654 knows the plan and is working towards it.

File: addons/aircraft/functions/fnc_showDialog.sqf [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+{
+    private _pylon = _x;
+    private _ctrl = _display ctrlCreate ["RscCombo", -1];
+
+    private _uiPos = getArray (_pylon >> "UIposition");
+    private _picturePos = ctrlPosition (_display displayCtrl 120);
+    _ctrl ctrlsetPosition [
+        (_picturePos select 0) + (_uiPos select 0),
+        (_picturePos select 1) + (_uiPos select 1),
+        0.100000 * safezoneW,
```
Comment (Neviothr) [inline addons/aircraft/functions/fnc_showDialog.sqf]: `0.100000` --> `0.1`?

**Comment by Tuupertunut** [comment]
@654wak654 What's your status on the integration with `rearm` component? I'm currently doing large fixes to rearming, including its pylon support. Have you changed something inside `rearm`? Have you encountered any bugs?

**Comment by 654wak654** [comment]
@Tuupertunut This little project is going to be in it's own module called `pylons`, which is going to be built with rearm's functions. Only local change I have so far is that I made the pylons module; haven't started to fiddle with rearm yet.

**Comment by dixon13** [comment]
I feel like this should be a feature of CBA maybe. I'm not sure. Anyone else have thoughts on this?

**Comment by jonpas** [comment]
> I feel like this should be a feature of CBA maybe. I'm not sure. Anyone else have thoughts on this?

I don't think so, it's perfect for ACE3, it's not a generalized framework or anything that CBA does, it's not even a framework.

**Comment by aristenn** [comment]
Does this being closed mean that we're not going to get this feature?

**Comment by sargken** [comment]
He didn't have time to finish it most likely

**Comment by jonpas** [comment]
There will be a new PR opened, since it has been redesigned that much it was easier to go from start.
Source: @654wak654

**Comment by jonpas** [comment]
Replaced by #5517.

**Pull request closed without merging**